app/graphql/mutations/account/create.rb

```diff
@@ -0,0 +1,42 @@
+class Mutations::Account::Create < Mutations::Base
+  include PublicMutation
+  include RateLimitedMutation
+  include FancyMutation
+
+  description 'Create a new Kitsu account'
+
+  rate_limit do
+    limit 20, per: 1.hour
+    limit 20, per: 2.days
+  end
+
+  input do
+    argument :name, String,
+      required: true,
+      description: 'The name of the user'
+
+    argument :email, String,
+      required: true,
+      description: 'The email address to reset the password for'
+
+    argument :password, String,
+      required: true,
+      description: 'The password for the user'
+
+    argument :external_identity, Types::Input::Account::ExternalIdentity,
+      required: false,
+      description: 'An external identity to associate with the account on creation'
+  end
+  result Types::Account
+  errors Types::Errors::Validation
```

Comment:
i'm thinking this should be called `union_errors` just to be explicit about it. When I saw this the first thing I asked was "isn't this for unions" and had to check the class.